package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/eth-ledger-bridge-keyring",
-  "version": "0.13.0",
+  "version": "0.14.0",
```

Comment:
Should this be 0.13.1 instead, since it only includes a fix?

Reply:
Done!

Reply:
changed back to 0.14.0